perceval/components/abstract_processor.py

```diff
@@ -35,9 +35,8 @@
 
```

Comment:
I think it will be useful to add some `debug` level logs for the method `_compose_processor()` (L263). Could you some there as well?

Sorry Github is not letting me put inline comments there.